engine/polyglot-api/src/main/java/org/enso/polyglot/RuntimeOptions.java

```diff
@@ -104,6 +104,16 @@ public class RuntimeOptions {
   private static final OptionDescriptor ENABLE_EXECUTION_TIMER_DESCRIPTOR =
       OptionDescriptor.newBuilder(ENABLE_EXECUTION_TIMER_KEY, ENABLE_EXECUTION_TIMER).build();
 
+  public static final String WARNINGS_LIMIT = optionName("warningsLimit");
+
+  @Option(
+      help = "Maximal number of warnings reported to the user.",
```

Comment:
```suggestion
      help = "Maximal number of warnings that can be attached to a value.",
```

Reply:
We can have more warnings reported, this is a per-value limit. Right? So IMO the description should reflect that.